lib/Conversion/TritonGPUToLLVM/TritonGPUToLLVM.cpp

```diff
@@ -1225,6 +1247,375 @@ struct BroadcastOpConversion
   }
 };
 
+/// ====================== reduce codegen begin ==========================
+
+struct ReduceOpConversion
+    : public ConvertTritonGPUOpToLLVMPattern<triton::ReduceOp> {
+public:
+  using ConvertTritonGPUOpToLLVMPattern<
+      triton::ReduceOp>::ConvertTritonGPUOpToLLVMPattern;
+
+  LogicalResult
+  matchAndRewrite(triton::ReduceOp op, OpAdaptor adaptor,
+                  ConversionPatternRewriter &rewriter) const override;
+
+private:
+  void doAcc(ConversionPatternRewriter &rewriter, Location loc, RedOp redOp,
+             Value &acc, Value cur, bool isFirst) const;
+
+  Value shflSync(ConversionPatternRewriter &rewriter, Location loc, Value val,
+                 int i) const;
+
+  Value stShared(ConversionPatternRewriter &rewriter, Location loc, Value ptr,
```

Comment:
storeShared?
The others might not be familiar with the `st` instruction, and might not get the meaning of `st`.

Reply:
Done.